### PR TITLE
Update teams-experience-o365odb-spo-multi-geo.md

### PR DESCRIPTION
**Pull request by cimares**
Added a link to the M365 roadmap item for Multi Geo support for Teams Chat, added as a result of enterprise demand.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-experience-o365odb-spo-multi-geo.md [hunk 1]
```diff
@@ -30,6 +30,6 @@ A given Team’s SharePoint site is also Multi-Geo-aware. That is, if a European
 
 Because a Multi-Geo tenancy is a single global tenant, during @ mentions satellite users will be able to see their colleagues from across the globe, no matter where they are.
 
-Conversations in chats, channels messages, and meeting IM notes/chats within the Teams experience aren't Multi-Geo-aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs. For more information, see [Location of data in Microsoft Teams](location-of-data-in-teams.md).
+Conversations in chats, channels messages, and meeting IM notes/chats within the Teams experience aren't Multi-Geo-aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs however as a result of demand from enterprise businesses, Multi-geo support is coming during 2021. You can [View the M365 Roadmap entry here](https://www.microsoft.com/en-gb/microsoft-365/roadmap?filters=&searchterms=70783) to track the anticipated timeline. For more information, see [Location of data in Microsoft Teams](location-of-data-in-teams.md).
```
Comment (JohanFreelancer9) [inline Teams/teams-experience-o365odb-spo-multi-geo.md:33]:
```suggestion
Conversations in chats, channels messages, and meeting IM notes/chats within the Teams experience aren't Multi-Geo-aware and are all kept only inside the central location of the tenant. Typically, chat conversations aren’t applied to data residency needs. However, as a result of demand from enterprise businesses, Multi-geo support is coming during 2021. You can [View the M365 Roadmap entry here](https://www.microsoft.com/en-gb/microsoft-365/roadmap?filters=&searchterms=70783) to track the anticipated timeline. For more information, see [Location of data in Microsoft Teams](location-of-data-in-teams.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@cimares - please commit my suggested changes if you agree. Thanks.

**Pull request merged**